packages/svelte/tests/compiler-errors/samples/runes-wrong-state-placement-2/main.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+	function a() {
+		return $state(1);
```

Comment:
what I like to do in these tests is also have another function above that passes, so that you know "this one doesn't throw, this one does throw, as expected"

Reply:
Thanks!